### PR TITLE
add layer design

### PR DESCRIPTION
**Pull request by AlaaAlamal2007**

### CODE REVIEW

File: src/main/java/com/example/alaa/university/UniversityApplication.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 public class UniversityApplication {
 
 	public static void main(String[] args) {
+
```
Comment (Andriireva) [inline src/main/java/com/example/alaa/university/UniversityApplication.java:10]: You can leave it. But this change it is not necessary

File: src/main/java/com/example/alaa/university/domain/University.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import java.util.List;
+
+public class University {
+    private Integer id;
+    private String name;
+    private Address address;
+    private UniversityType universityType;
+    private  String email;
+    private  Double studyCost;
+    private Instant startStudyDay;
```
Comment (Andriireva) [inline src/main/java/com/example/alaa/university/domain/University.java:13]: According to the name it looks like it should be in Student Class. Maybe you wanted that date be "foundAt" ? so it means when this University was found, and started operating

File: src/main/java/com/example/alaa/university/domain/University.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import java.time.Instant;
+import java.util.List;
+
+public class University {
+    private Integer id;
+    private String name;
+    private Address address;
+    private UniversityType universityType;
+    private  String email;
+    private  Double studyCost;
```
Comment (Andriireva) [inline src/main/java/com/example/alaa/university/domain/University.java:12]: it is suggested to remove unnecessary spaces. You can use ctrl + alt + l that format the code

File: src/main/java/com/example/alaa/university/domain/University.java (hunk 2 above)
Comment (AlaaAlamal2007) [reply src/main/java/com/example/alaa/university/domain/University.java:13]: yes ,I  mean started operating

File: src/main/java/com/example/alaa/university/domain/University.java (hunk 3 above)
Comment (AlaaAlamal2007) [reply src/main/java/com/example/alaa/university/domain/University.java:12]: ok,

**Review by Andriireva — APPROVED** [review]

**Pull request merged**